check-code/Makefile

```diff
@@ -0,0 +1,5 @@
+test: .cabal-sandbox
```

Comment:
I _think_ so as the `.cabal-sandbox` rule produces that directory. I think what you would want is to have that rule not only depend on `tinc.yaml` but also on the hackage index on your local machine, so that `tinc` would be run again when the hackage index changes. But I'm not sure that can be done.